easybuild/easyconfigs/p/pycocotools/pycocotools-2.0.1-fosscuda-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'pycocotools'
+version = '2.0.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/cocodataset/cocoapi'
+description = "Tools for working with the MSCOCO dataset"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['1c06e73a85ed9874c1174d47064524b9fb2759b95a6997437775652f20c1711f']
+
+dependencies = {
```

Comment:
a set also works but this is typically a list :)

Reply:
well spotted :)
fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/11211/commits/f311672ece41811e22d332d35168bdf3395c0f1b